pkg/resources/function_commons.go

```diff
@@ -214,6 +230,11 @@ func functionBaseSchema() map[string]schema.Schema {
 						DiffSuppressFunc: DiffSuppressDataTypes,
 						Description:      "The argument type.",
 					},
+					"arg_default_value": {
```

Comment:
I think we could drop `arg_` prefix, in other fields as well, wdyt?

Reply:
Yes, I will do it.